easybuild/easyconfigs/t/timm/timm-0.9.2-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonBundle'
+
+name = 'timm'
+version = '0.9.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://huggingface.co/docs/timm'
+description = """
+timm is a library containing SOTA computer vision models, layers, utilities,
+optimizers, schedulers, data-loaders, augmentations, and training/evaluation
+scripts.  It comes packaged with >700 pretrained models, and is designed to be
+flexible and easy to use.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('PyTorch', '1.12.0', versionsuffix),
+    ('PyYAML', '6.0'),
+    ('tqdm', '4.64.0'),
+    ('torchvision', '0.13.1', versionsuffix),
+]
+
+builddependencies = [
+    ('Rust', '1.65.0'),  # for safetensors
+]
+
+use_pip = True
+
+exts_list = [
+    ('huggingface_hub', '0.15.1', {
+        'checksums': ['a61b7d1a7769fe10119e730277c72ab99d95c48d86a3d6da3e9f3d0f632a4081'],
+    }),
+    ('safetensors', '0.3.1', {
+        'checksums': ['571da56ff8d0bec8ae54923b621cda98d36dcef10feb36fd492c4d0c2cd0e869'],
+    }),
```

Comment:
CargoPythonPackage should work here. Even if just defaulting to fetching deps online it's still beneficial to use this easyblock, as it protects your home dir from CARGO_HOME and optimizes march=native